.github/workflows/ci.yml

```diff
@@ -207,6 +208,12 @@ jobs:
           git switch -c $BRANCH
           git push -u origin $BRANCH
       - name: Create Pull Request
-        run: gh pr create -B master --title 'Automatic sync from rustc' --body ''
+        run: |
+          PR=$(gh pr create -B master --title 'Automatic sync from rustc' --body '')
```

Comment:
Looking at https://github.com/rust-lang/miri/actions/runs/5441198364/jobs/9894950685, `gh` exactly outputs the URL, so this should work - hopefully. :)